Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1526 @@
+            if original_obj_action == object_action:
+                object_values.extend(
+                    objects["object_type"][object_type].get(object_action, [])
+                )
+            else:
+                original_obj_actions[original_obj_action] = [
+                    value
+                    for value in original_obj_actions[original_obj_action]
+                    if value not in object_values
+                ]
+
+        objects["object_type"][object_type].update({object_action: object_values})
+
+    elif any([object_type, object_action, object_values]):
+        raise DemistoException(
+            "object_type, object_action, object_values should be used in conjunction."
+        )
+
+    if block_custom_mime_types:
+        objects["block_custom_mime_types"] = block_custom_mime_types
+
+    objects["max_object_size_mb"] = remove_empty_elements(
+        {
+            "http_or_https": http_or_https_max_object_size_mb,
+            "ftp": ftp_max_object_size_mb,
+        }
+    )
+
+
+def access_policy_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    List access policies.
+
+    Args:
+        client (Client): Cisco WSA API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: readable outputs for XSOAR.
+    """
+    policy_names = args.get("policy_names")
+    response = client.access_policy_list_request(policy_names=policy_names).get(
+        "access_policies", []
+    )
+
+    paginated_response = pagination(response=response, args=args)
+
+    readable_output = tableToMarkdown(
+        name="Access Policies",
+        t=paginated_response,
+        headers=[
+            "policy_name",
+            "policy_status",
+            "policy_order",
+            "policy_description",
+            "policy_expiry",
+        ],
+        headerTransform=string_to_table_header,
+        removeNull=True,
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="CiscoWSA.AccessPolicy",
+        outputs_key_field="policy_name",
+        outputs=paginated_response,
+        raw_response=paginated_response,
+    )
+
+
+def access_policy_create_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    policy_name = args["policy_name"]
+    policy_status = args["policy_status"]
+    policy_order = arg_to_number(args["policy_order"])
+    identification_profile_name = args["identification_profile_name"]
+    policy_description = args.get("policy_description")
+    policy_expiry = args.get("policy_expiry")
+
+    response = client.access_policy_create_request(
+        policy_name=policy_name,
+        policy_status=policy_status,
+        policy_order=policy_order,
+        identification_profile_name=identification_profile_name,
+        policy_description=policy_description,
+        policy_expiry=policy_expiry,
+    )
+
+    if response.status_code == 204:
```

Comment:
```suggestion
    if response.status_code == HTTPStatus.NO_CONTENT:
```
You can use: `from http import HTTPStatus`